product_docs/docs/biganimal/release/using_cluster/managing_replicas.mdx

```diff
@@ -0,0 +1,104 @@
+
+1. Sign in to the [BigAnimal](https://portal.biganimal.com/) portal.
+
+1. Go to the [Clusters](https://portal.biganimal.com/clusters) page. A list of previously created clusters appears.
+
+1. Select the *Faraway Replica* icon next to a cluster to create its faraway replica. The Create Replica page appears.
+
+1. You cannot edit the **Cloud Provider** field. The cloud provider is inherited from the parent cluster. 
+
+1. In the Replica Settings tab, Enter a name for your replica in the **Faraway Replica Name** field.
+
+1. Enter a password for your replica in the **Password** field.
+
+1. You cannot edit the **Database Type** field. The Postgres type and Postgres version are inherited from the parent cluster.  
+
+1. In the **Region** section, select the region where you want to deploy your replica.
+
+ You can select a region for deploying your replica even if it is not yet *Active*. Your replica creation request is added to a queue, and the replica is created after you activate the region. See [Activating regions](/biganimal/latest/getting_started/activating_regions) for more information.
+ 
+1.  The fields in the **Instance Type** section are prepopulated with values inherited from the parent cluster and are editable. You can edit these fields. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on instance type settings.
+
+1.  The fields in the **Storage** section are prepopulated with values inherited from the parent cluster. You can edit these fields. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the storage settings.
+ 
+1.  In the **Networking** section, specify whether to use private or public networking. See [Creating a cluster](/biganimal/latest/getting_started/creating_a_cluster/#cluster-settings-tab) for details on the Networking settings.
+
+1. Select **Create Replica**. 
+
+
+### Promote a faraway replica
+
+You can promote a faraway replica to a full-fledged cluster making it capable of accepting writes. The new cluster created is different from the parent cluster and it does not replace the parent cluster.
+
+1. Sign in to the [BigAnimal](https://portal.biganimal.com/) portal.
+
+1. Go to the [Clusters](https://portal.biganimal.com/clusters) page. A list of previously created clusters appears.
+
+1. Select the cluster with the replica you want to promote.
+
+1. In the **Overview** tab, you can see the cluster's replicas under **Faraway Replicas**.
+
+1. Select the *Promote replica* icon next to the replica you want to promote. The Create Cluster page appears. 
+
+1. The cluster settings are prepopulated with values inherited from the parent cluster. You can edit the cluster settings while creating your cluster. 
+
+   !!! NoteNotes
+     - A faraway replica can be promoted to a single node cluster or a high availablility cluster, but not an extreme high availability cluster. 
```

Comment:
Make this active voice: You can promote a faraway replica to a single node or high availability cluster but not to an extreme high availability cluster.
No comma was needed before "but" since what followed the but was not a complete sentence.

Reply:
Changed.